src/ciphers/morse_code.rs

```diff
@@ -1,8 +1,35 @@
 use std::collections::HashMap;
 
+// The character used to represent an unknown morse code sequence
 const UNKNOWN_CHARACTER: &str = "........";
+
+// The character used to represent an unknown morse code character
 const _UNKNOWN_MORSE_CHARACTER: &str = "_";
 
+/// Encode a message into morse code.
+///
+/// Given a message, this function encodes it into morse code.
+/// It uses a dictionary to map each character to its corresponding morse code sequence.
+/// If a character is not found in the dictionary, it is replaced with the unknown character sequence.
+///
+/// # Arguments
+///
+/// * `message` - The message to encode into morse code.
+///
+/// # Returns
+///
+/// The encoded morse code as a string.
+///
+/// # Examples
+///
+/// ```rust
+/// use rust_algorithms::ciphers::encode;
+///
+/// let message = "Hello Morse";
+/// let cipher = encode(message);
+///
+/// assert_eq!(cipher, ".... . .-.. .-.. --- / -- --- .-. ... .");
+/// ```
```

Comment:
Since you added this as a doc test, could you delete the corresponding test in `mod tests`?